tests/next_tests/unit_tests/ffront_tests/test_func_to_past.py

```diff
@@ -206,7 +206,9 @@ def domain_format_3_program(in_field: gtx.Field[[IDim], float64]):
     assert exc_info.match("Invalid call to `domain_format_3`")
 
     assert (
-        re.search("Missing required keyword argument\(s\) `out`.", exc_info.value.__cause__.args[0])
+        re.search(
```

Comment:
Just a question: why has the format of this line been changed? It looks exactly the same as the change below which kept the same format...